helpdesk/templates/helpdesk/ticket.html

```diff
@@ -44,6 +44,13 @@
         $(wrapper).append("<div><label class='btn btn-primary btn-sm btn-file'>Browse... <input type='file' name='attachment' id='file" + x + "' multiple style='display: none;'/></label><span>&nbsp;</span><span id='selectedfilename" + x + "'>{% trans 'No files selected.' %}</span></div>"); //add input box
     });
 
+    {% if not is_staff %}
+    // If the user is not staff, they should not have access to all operations
```

Comment:
Some what pedantic, but this comment won't show unless the block does, and then contextually it might not make sense.  I think we could use {# comment #}.  Maybe that was not the intent of the author.  Just a consideration.